openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/ActivityFeedTab.component.tsx

```diff
@@ -256,6 +283,20 @@ export const ActivityFeedTab = ({
     };
   }, [activeTab, isUserEntity, currentUser]);
 
+  const handleSaveAnnouncement = useCallback(() => {
+    getFeedData(
+      feedFilter,
+      undefined,
+      ThreadType.Announcement,
+      entityType,
+      fqn,
+      undefined,
+      undefined,
+      announcementFilter
+    );
+    handleCloseAnnouncementModal();
```

Comment:
Why save method has getFeedData?

Reply:
To get the updated list of announcements